GameData/masTerTorch/kdex.version

```diff
@@ -12,7 +15,7 @@
         "MAJOR":1,
```

Comment:
Should possibly increase the major version here as this breaks compatibility with past versions of KSP.
Or at least increase minor version to V1.12

Reply:
agree. I just did a minor bump to communicate it to you; 👴  🏫

Reply:
Fair enough, thanks :)